File: examples/java/advanced/error-handling/test-sdk-validation.java
@Test
void testSdkValidationError() {
    var handler = new InvalidConfigHandler();
    var runner = LocalDurableTestRunner.create(Map.class, handler);

    var result = runner.runUntilComplete(Map.of());

    // SDK should catch invalid configuration
    assertEquals(ExecutionStatus.FAILED, result.getStatus());
    assertTrue(result.getError().getMessage().contains("IllegalArgumentException"));
}

File: examples/python/sdk-reference/error-handling/test-sdk-validation.py
@pytest.mark.durable_execution(
    handler=handler_with_invalid_config,
    lambda_function_name="sdk_validation_function",
)
def test_sdk_validation_error(durable_runner):
    """Test SDK validation error handling."""
    with durable_runner:
        result = durable_runner.run(input={}, timeout=10)
    
    # SDK should catch invalid configuration
    assert result.status is InvocationStatus.FAILED
    assert "ValidationError" in str(result.error)
